packages/rum-core/src/domain/rumEventsCollection/view/trackViews.spec.ts

```diff
@@ -208,6 +209,87 @@ describe('rum track url change', () => {
   })
 })
 
+describe('rum use onNewLocation callback to rename/ignore views', () => {
```

Comment:
Could you add a test for the `oldLocation` parameter? (check that it is undefined for initial view, and defined with a correct value for other views)